.gitignore

```diff
@@ -14,6 +14,10 @@ node_modules/
 .env.development.local
 .env.test.local
 .env.production.local
+.env
+
+# Gmail secrets
+gmail-secret.json
```

Comment:
we eventually will want to make this an environment variable - so we can deploy. There are a couple of ways to do that, one easy way is to base64 encode the json, and then decode it as json from the env variable when we need to. 

this will be something we can add to the icebox (as a ticket) - if you can do that, that would be great

Reply:
Did not know you could do that. Ticket added. Thanks